### ISSUE TITLE
Gallery Display

### ISSUE THREAD

**Issue by poluramus** [11cafe/comfyui-workspace-manager#357]
Hello!

I'd like to start by highlighting your extension, which stands out from others I've found. While many extensions offer partial solutions, yours covers everything I need. Thank you for your great work!

Next, I have a question: is it possible to display images based on their folder structure? Currently, the images are organized by workflow, but a traditional folder view with basic metadata would be a great addition. May-be from that view, there could be a redirection to the image-worflow image.

Thank you!

**Comment by aphaits** [comment]
Was just looking for this, Would add another suggestion is to fork this feature into its own release that can explore desktop folders.

**Comment by Weixuanf** [comment]
thanks for your recognition. I'm really glad the gallery feature is useful to you! ☺️

> it possible to display images based on their folder structure?  .. a traditional folder view with basic metadata would be a great addition. 

this is a nice idea. I was planning to show all your images in gallery, but haven't got to it yet. So I'm thinking of adding "All Images" tab that upon click, you will see all the images in folder view, after click an image, you will see the "image - metadata form" to inspect the image metadata.

**Comment by Weixuanf** [comment]
> Was just looking for this, Would add another suggestion is to fork this feature into its own release that can explore desktop folders.

oh you mean a desktop standalone app to inspect images metadata?

**Comment by aphaits** [comment]
> > Was just looking for this, Would add another suggestion is to fork this feature into its own release that can explore desktop folders.
> 
> oh you mean a desktop standalone app to inspect images metadata?

Yes please! It would be a great comfyui or general SD based image browser / annotator.
I can already imagine several gallery only feature such as favorite image / prompt, or maybe browse based on checkpoint filter / keyword filter. Maybe also ability to scan a folder / multiple folders and then list out most used keywords / checkpoints / lora / setting.

It would be a great reference tool.
